### DISCUSSION TITLE
v1/zh-hans/docs/look-and-feel/

### DISCUSSION THREAD

**Discussion by giscus** [razonyang/hugo-theme-bootstrap#966] [General]
# v1/zh-hans/docs/look-and-feel/

默认情况下，使用此主题的站点具有默认字体、颜色和总体外观。但是，默认方案不可能满足所有人的要求，但是不要担心，你可以轻松地覆盖主题默认值，例如调色板颜色、字体、语法高亮。

https://hbs.razonyang.com/v1/zh-hans/docs/look-and-feel/

**Comment by phenix3443** [comment]
我刚刚用这个主题建立了一个自己的[blog](https://blog.panghuli.cn/), 当时右上角的配色模块（palette-dropdown-menu）不能正常展示，显示一个黑色或者白色的小方块，方便帮忙看下是什么原因么？

**Reply by phenix3443** [reply]
我刚刚用这个主题建立了一个自己的blog, 但是右上角的配色模块（palette-dropdown-menu）不能正常展示，显示一个黑色或者白色的小方块，方便帮忙看下是什么原因么？

**Reply by razonyang** [reply]
应该和 Bootstrap v5.3.1 的改动有关，我待会修改下。

**Reply by razonyang** [reply]
@phenix3443 [升级主题](https://hbs.razonyang.com/v1/zh-hans/docs/getting-started/upgradation/)到 `v1.2.2` 应该就好了。

**Reply by phenix3443** [reply]
可以了，正常了，另外关于 action panel可以写文章说明一下么？为什么会有这个功能？主要的使用场景是什么？除了 facebook 和 twitter 还有别的可以扩展的选项么？

**Reply by razonyang** [reply]
Action panel 一般用于 PWA，安装后界面如下，少了导航栏，而 Action panel 用于补充这块操作。

![image](https://github.com/razonyang/hugo-theme-bootstrap/assets/17720932/0abf09fd-8f64-46e1-8e69-cd231c98b4e3)

社交分享暂时没有实现其社交媒体的计划，欢迎提交 PR，对应的源码：https://github.com/razonyang/hugo-theme-bootstrap/blob/38321ed9bb3e68bd6ba47a701a8febb6293ff5b6/layouts/partials/footer/actions-panel.html 。

另外如果不需要这个，也可以通过配置禁用，另请参阅 https://hbs.razonyang.com/v1/zh-hans/docs/configuration/site-params/ 。
